DEI leadership post (#267)
2i2c-org/2i2c-brochure: content/blog/2024/dei-leadership-strategy/index.md

However, over the past few months, we have reflected on our organization's structure and future plans and noticed a worrying lack of women in staff leadership roles. This imbalance does not live up to our values of diversity in our team and service.

We need to do better.

2i2c is at a moment of maturation and growth as an organization, kicked off by [our organizational audit from 2023](../2023/organizational-report/index.md) as well as [our three-year retrospective](../report-czi/index.md).
We believe that improving the diversity of leadership throughout the organization is a necessary part of that maturation over the next three years.

Below are a few ideas for how we aim to make improvements, and we invite feedback from others who are interested in helping us improve this aspect of our organization.

## What we plan to do

Over the next few months, we will work on a long-term DEI strategy in tandem with our sustainability strategy as we work [towards scalable and sustainable networks of community hubs](../funding-community-networks/index.md).

Here are some steps we aim to take to improve the diversity of our leadership across the organization:

> 1. **Define our goals for diversity**. First, we need a better understanding of the most important axes of diversity that we wish to design around. Gender is clearly a critical gap to cover, but there are many other important axes of diversity as well. For example, as an organization that serves a global community, it is important that we have global perspectives represented in our leadership as well.

Reviewer: Remove repeated "as well"
```suggestion
1. **Define our goals for diversity**. First, we need a better understanding of the most important axes of diversity that we wish to design around. Gender is clearly a critical gap to cover, but there are many other important axes of diversity as well. For example, as an organization that serves a global community, it is important that we have global perspectives represented in our leadership.
```

> 2. **Define mechanisms for leadership and governance that includes representation along these axes**. Leadership at a staff level is one way to ensure representation of diverse perspectives, but there are many other ways to bring voices into the conversation as well. We aim to explore new ways of bringing diverse voices into the strategic direction of the organization and provide mechanisms for holding us accountable to our values. For example, we intend to grow a _board_ to guide 2i2c's strategy and hold its Executive Director accountable for achieving impact - this is another opportunity to bring diverse perspectives into our leadership.

Reviewer: ```suggestion
2. **Define mechanisms for leadership and governance that includes representation along these axes**. Leadership at a staff level is one way to ensure representation of diverse perspectives, but there are many other ways to bring voices into the conversation. We aim to explore new ways of bringing diverse voices into the strategic direction of the organization and provide mechanisms for holding us accountable to our values. For example, we intend to grow a _board_ to guide 2i2c's strategy and hold its Executive Director accountable for achieving impact - this is another opportunity to bring diverse perspectives into our leadership.
```

## The importance of diverse _leadership_

> However, over the past few months, we have reflected on our organization's structure and future plans and noticed a worrying lack of women in staff leadership roles. This imbalance does not live up to our values of diversity in our team and service.

Reviewer: @choldgraf very minor point -- "noticed" implies it's suddenly occurred to us that there are very few women in the organisation. I don't feel confident that this accurately describes our position and how we got here. Rather, my assumption is that Jenny's original wording here better speaks to the fact that it's clearly an issue, but we have yet to solve it. This post, to my eyes, is saying "we need to solve it now". 

Please, _don't let me speak over more qualified voices on this subject_ (I have probably not been in any of these conversations), and _reject this if you think I've missed the point_, but I'll suggest a reword?

```suggestion
However, over the past few months, we have reflected on our organization's structure and future plans. In particular, we recognize that there is a worrying lack of women in staff leadership roles. This imbalance does not live up to our values of diversity in our team and service.
```

Reviewer 2: That works for me - this is something I've been stressed out about for a long time. I think that the slide deck sharpened attention on this point, as opposed to made it understood for the first time.

4. **Continue improving our systems of delivery and work**. Finally, we believe that a crucial aspect of improving the diversity in our organization is to continue transforming our systems for delivery, reliability, and accountability across the organization. This effort is a crucial step for fostering an inclusive and equitable culture. We have [made a lot of progress in improving our delivery systems](../delivery-improvements/index.md) and will continue this improvement as an organization.

We hope that this post both makes transparent a distinct lack of diversity within our organizational leadership and provides clarity for some of our plans to improve.

> We are excited about the organization that we have built so far, as well as being fiercely proud of our open-source culture and development practises.

Reviewer: > fiercely proud

:heart_eyes:

Author: I'm pretty sure our whole team feels this way ❤️‍🔥
